src/store/modules/job/actions.ts

```diff
@@ -437,6 +437,32 @@ const actions: ActionTree<JobState, RootState> = {
       const tempExpr = Object.values(cached).map((job: any) => job.tempExprId)
       await dispatch('fetchTemporalExpression', tempExpr)
 
+      cached = Object.values(cached).map((cachedJob: any) => ({
```

Comment:
I think this will convert the cached into an array and thus we will not be able to fetch the job data from the cached state.

Reply:
Right sir! I am fixing the logic.